src/app/editor/primitive-field/primitive-field.component.html

```diff
@@ -1,19 +1,20 @@
 <div [ngSwitch]="valueType">
   <div>
     <div *ngSwitchCase="'string'">
-      <textarea rows="1" [ngModel]="value" (ngModelChange)="onModelChange($event)"></textarea>
+      <textarea rows="1" [ngModel]="value" (ngModelChange)="onModelChange($event)" placeholder="{{schema.title}}" disabled="{{schema.x_editor_disabled}}"></textarea>
     </div>
     <div *ngSwitchCase="'enum'">
+      <!-- TODO: set placeholder -->
       <searchable-dropdown [value]="value" [items]="schema.enum" (onSelect)="onModelChange($event)"></searchable-dropdown>
     </div>
     <div *ngSwitchCase="'autocomplete'">
-      <autocomplete-input [value]="value"  [autocompletionOptions]="schema.x_editor_autocomplete" (onValueChange)="onModelChange($event)"></autocomplete-input>
+      <autocomplete-input [value]="value"  [autocompletionOptions]="schema.x_editor_autocomplete" (onValueChange)="onModelChange($event)" [placeholder]="schema.title"></autocomplete-input>
     </div>
```

Comment:
`disabled` is not set for `searchable-dropdown` and `autocomplete-input` because I think we should be disabling the fields that are autocomplete or enum.